docs/settings/alert-action-settings.asciidoc

```diff
@@ -188,10 +188,14 @@ For example, `20m`, `24h`, `7d`, `1w`. Default: `60s`.
 ==== Alerting settings
 
 `xpack.alerting.maxEphemeralActionsPerAlert`::
-Sets the number of actions that will be executed ephemerally. To use this, enable ephemeral tasks in task manager first with <<task-manager-settings,`xpack.task_manager.ephemeral_tasks.enabled`>>
+Sets the number of actions that will run ephemerally. To use this, enable
+ephemeral tasks in task manager first with
+<<task-manager-settings,`xpack.task_manager.ephemeral_tasks.enabled`>>
 
 `xpack.alerting.cancelAlertsOnRuleTimeout`::
-Specifies whether to skip writing alerts and scheduling actions if rule execution is cancelled due to timeout. Default: `true`. This setting can be overridden by individual rule types.
+Specifies whether to skip writing alerts and scheduling actions if a rule did
+not run due to a timeout. Default: `true`. This setting can be overridden by
```

Comment:
Nit: This should not be `if a rule did not run due to timeout` since the rule ran but its run was cut short due to timeout. Not sure the best way to word that is.

Reply:
Looking at the phrasing you used in https://github.com/elastic/kibana/pull/131459, IMO this could be:

> Specifies whether to skip writing alerts and scheduling actions if rule processing was cancelled due to a timeout.
 
> Specifies whether to skip writing alerts and scheduling actions when a  rule run times out.

WDYT?

Reply:
Yea, I think either of those would work!

Reply:
Thanks! I implemented the first one in 835953a555ad12f64e2435ce44195db153e4a67c